docs/manual/spec.md

```diff
@@ -476,6 +476,25 @@ If the main section exists, it must come first to avoid ambiguity.
 Otherwise, append and prepend can be used in any order and multiple
 times, even if the corresponding main section does not exist.
 
+During the execution of build scriptlets, (at least) the following
+rpm-specific environment variables are set:
+
+Variable            | Description
+---------------------------------------------------
+RPM_ARCH            | Architecture of the package
+RPM_BUILD_DIR       | The build directory of the package
+RPM_BUILD_NCPUS     | The number of CPUs available for the build
+RPM_BUILD_ROOT      | The buildroot directory of the package
+RPM_BUILD_TIME      | The build time of the package
```

Comment:
This should specify format/units, i.e. seconds since since the epoch.

Reply:
Good point, thanks. Fixed in the latest push.